test/test_ops.py

```diff
@@ -231,69 +231,63 @@ def test_variant_consistency_jit(self, device, dtype, op):
         for sample in samples:
 
             # Acquires variants to test
+            func = op.get_op()
             method = op.get_method()
             inplace = op.get_inplace()
-            variants = (v for v in (method, inplace) if v is not None)
-
-            # Adds function variant to variant list
-            # TODO: inplace tests currently fail
-            # variants = (v for v in (op, method, inplace) if v is not None)
-            variants = (v for v in (op, method) if v is not None)
+            variants = {
+                'function': func, 'method': method,
+                # TODO: inplace tests currently fail
+                # 'inplace': inplace,
+            }
 
             # Test traced and scripted consistency
-            for variant in variants:
+            for func_type, variant in variants.items():
+                if variant is None:
+                    continue
+
                 # Create accessor for script function variant
-                if variant is op:
-                    name = op.name
-                    func_type = 'function'
-                elif variant is method:
-                    name = op.name
-                    func_type = 'method'
-                else:  # variant is inplace
-                    assert variant is inplace
-                    name = op.name + "_"
-                    func_type = 'inplace'
+                name = op.name + '_' if func_type == 'inplace' else op.name
 
                 # run with disable_autodiff_subgraph_inlining(True) to test
                 #   autodiff support. Context manager forces the graph to contain
                 #   DifferentiableGraph nodes if they are present
                 with disable_autodiff_subgraph_inlining():
                     def fn(*inputs, **kwargs):
-                        attr = getattr(inputs[0], name)
-                        output = attr(*inputs[1:], **kwargs)
+                        output = func(*inputs, **kwargs)
```

Comment:
Again, failed because there is no method variant. Then again, there are methods without function variants as well. Would it make sense to use `variant` here instead?

Reply:
On second thought, `get_op()` wouldn't return anything if there was no function variant. Is `OpInfo` meant to support that case?

Reply:
OpInfos weren't really designed to handle methods that don't have corresponding functions, and the few cases where this occurs in PyTorch probably just need a function variant to be added. 

This line originally requiring the method defined on torch.Tensor is probably just a historical oddity, and changing it to call the function-variant (and assuming there always is a function variant) is fine.